dask/dataframe/core.py

```diff
@@ -529,6 +529,22 @@ def clip(self, lower=None, upper=None):
     def notnull(self):
         return map_partitions(pd.Series.notnull, self.name, self)
 
+    def apply(self, func, convert_dtype=True, args=(), **kwds):
+        """ Parallel version of pandas.Series.apply
+
+        This is identical to the Pandas algorithm but assumes that axis=0.
```

Comment:
This seems like a bad idea. You don't need to support `axis=1` but you shouldn't change the default value.